examples/server/server.cpp

```diff
@@ -264,6 +265,21 @@ struct worker {
                 task->success = true;
                 response_map->push(task);
                 break;
+            case VOICES:
+                if (!runner->supports_voices) {
+                    task->message = "Voices are not supported for architecture '" + runner->arch_name() + "'.";
+                    response_map->push(task);
+                    break;
+                }
+                for (auto voice : list_voices(runner)) {
+                    if (!task->message.empty()) {
+                        task->message += ",";
```

Comment:
We really shouldn't use the message output for passing data back from the workers.